Services/UI/OJS.Services.Ui.Business/Extensions/SubmissionSerializationExtensions.cs

```diff
@@ -0,0 +1,27 @@
+namespace OJS.Services.Ui.Business.Extensions;
+
+using FluentExtensions.Extensions;
+using OJS.Services.Common.Models.Submissions.ExecutionContext;
+
+public static class SubmissionSerializationExtensions
+{
+    public static string ToSerializedDetails(this SubmissionServiceModel model)
+    {
+        var modelCopy = model.DeepCopy();
+
+        // These properties hold possibly big amount of information that we don't want copied in multiple db tables
+        modelCopy.FileContent = null;
+        modelCopy.Code = null;
+
+        modelCopy.TestsExecutionDetails!.Tests = modelCopy
+            .TestsExecutionDetails!
```

Comment:
Can you make it safer, without using `!`. If the properties are null, we will just not include them in the serialized data.